### ISSUE TITLE
Add slide show at client  section

### ISSUE THREAD

**Issue by Burrasupriya** [Pallavi1926/Gym_Website__Using_HTML_CSS_JAVASCRIPT#260]
I want to add a slide show at client section or if we have more clients then when we click on clients it will take to new page and our clients are displayed in cards

**Comment by rahulkumarparida** [comment]
# Client Section 
**I want to add the slide show to the project will submit it within 5hrs after assigning**

**Comment by Burrasupriya** [comment]
> # Client Section
> **I want to add the slide show to the project will submit it within 5hrs after assigning**

I can easily do it in below 1hr because i have worked on it previously in my internship

**Comment by rahulkumarparida** [comment]
> > # Client Section
> > **I want to add the slide show to the project will submit it within 5hrs after assigning**
> 
> I can easily do it in below 1hr because i have worked on it previously in my internship

Yeah Good for you. Also have you cloned and seen the website locally it's terrible someone messed the code up using grids and is looking terrible now gotta fix everything you know.
Am a B.tech 2nd year 🤧🤧(No Internship)
